.shippable.yml

```diff
@@ -6,13 +6,28 @@ env:
     global:
         - ZEPHYR_SDK_INSTALL_DIR=/opt/sdk/zephyr-sdk-0.11.3
         - ZEPHYR_TOOLCHAIN_VARIANT=zephyr
-        - MATRIX_BUILDS="5"
+        - MATRIX_BUILDS="20"
     matrix:
         - MATRIX_BUILD="1"
         - MATRIX_BUILD="2"
         - MATRIX_BUILD="3"
         - MATRIX_BUILD="4"
         - MATRIX_BUILD="5"
+        - MATRIX_BUILD="6"
+        - MATRIX_BUILD="7"
+        - MATRIX_BUILD="8"
+        - MATRIX_BUILD="9"
+        - MATRIX_BUILD="10"
+        - MATRIX_BUILD="11"
+        - MATRIX_BUILD="12"
+        - MATRIX_BUILD="13"
+        - MATRIX_BUILD="14"
+        - MATRIX_BUILD="15"
+        - MATRIX_BUILD="16"
+        - MATRIX_BUILD="17"
+        - MATRIX_BUILD="18"
+        - MATRIX_BUILD="19"
+        - MATRIX_BUILD="20"
```

Comment:
These changes of course needs to be removed before merging, otherwise looks good.